src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * A class that handles the file containing data regarding tasks.
+ *
+ * @author Toh Wang Bin
+ */
+public class Storage {
+
+    //path of file containing stored data
+    private String filePath;
+    //path of folder containing data file
+    private String directoryPath;
+    //FIle containing stored data
```

Comment:
Is this a typo with FIle? But I like how you explain what each field represents!